decidim-process-extended/Rakefile

```diff
@@ -10,25 +10,16 @@ require "rdoc/task"
 
 RDoc::Task.new(:rdoc) do |rdoc|
   rdoc.rdoc_dir = "rdoc"
-  rdoc.title = "Decidim::Process::Extended"
+  rdoc.title = "Decidim::Type"
```

Comment:
Why this title change? I don't understand it because the Rakefile is for decidim-process-extended.

Reply:
Too much copypasta! good catch!